astroplan/plots/time_dependent.py

```diff
@@ -145,40 +147,49 @@ def plot_airmass(target, observer, time, ax=None, style_kwargs=None,
                       'to use a scalar. (Or maybe a list with length > 1?).',
                       PlotWarning)
 
-    # Calculate airmass
-    airmass = observer.altaz(time, target).secz
-    # Mask out nonsense airmasses
-    masked_airmass = np.ma.array(airmass, mask=airmass < 1)
+    if not isinstance(targets, list):
```

Comment:
Yes, that probably is cleaner. Any preference between the two? It assumes it is a sequence of targets and I don't really see someone passing their own iterator.